css/desktop-style.css

```diff
@@ -0,0 +1,21 @@
+.nav a {
+  color: var(--neutrals-n-600);
+  text-align: center;
+  font: 500 0.9375rem/1.25 Poppins, sans-serif;
+}
+
+.intro {
+  background: url(../images/header.svg) var(--neutrals-n-0);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.intro-hero h1 {
+  max-width: 25rem;
+  letter-spacing: 0.02313rem;
+  font: 700 3rem/3.75rem Poppins, sans-serif;
+}
+
+.intro-hero p {
+  font: 400 1.25rem/1.75rem Poppins, sans-serif;
+}
```

Comment:
Nice work @IsaMarvin, your design is really looking good. However, the idea of this project is a mobile-first design. 
In a mobile-first design approach, typically a single stylesheet is used to handle the styling of the website or application across all devices. This single stylesheet is designed to be responsive, meaning it adapts the layout and styles based on the screen size and device being used and then you have to used media queries to define the styles for the desktop version instead of creating several stylesheets as you have done. 
Kindly update your project accordingly since this is part of the project requirements as highlighted below. :point_down: 

![Screenshot from 2023-07-14 18-36-39](https://github.com/IsaMarvin/Portfolio_Microverse/assets/65233860/e5400c2a-91b4-4504-95be-b05727eb9155)